pages/news.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <v-container>
-    <v-row>
+    <v-row> 
+      <v-sheet v-if="notLoaded" class="mx-3">
+        <h1 class="mt-3">
+          {{ $t('news.title') }}
+        </h1>
+        <p class="mb-6">
+          {{ $t('news.blurb') }}
+        </p>
+        <v-skeleton-loader type="card-avatar, article" dark="true" elevation="2" min-width="1160" class="my-5" height="400" /> 
```

Comment:
```suggestion
        <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-5" height="400"></v-skeleton-loader> 
```
- For those boolean properties in vuetify components, you can simply specify `dark` instead of `dark="true"`. But I think in this case, looks like your intention was not to use it?
- The `min-width` can be removed as it overextends if user has a smaller screen (ie. mobile).
- Although it works, it's good practice to use the self closing tags only for very specific html elements that allow it in the HTML5 rules. Ie: `<img />`, `<br />`. So in this case, just use `</v-skeleton-loader>`.